photonVision/cameras/Microsoft_LifeCam_HD-3000/pipelines/Cargo Red.json

```diff
@@ -0,0 +1,86 @@
+[ "ColoredShapePipelineSettings", {
+  "pipelineIndex" : 1,
+  "pipelineType" : "ColoredShape",
+  "inputImageFlipMode" : "NONE",
+  "inputImageRotationMode" : "DEG_0",
+  "pipelineNickname" : "Cargo Red",
+  "cameraExposure" : 49.1,
+  "cameraBrightness" : 50,
+  "cameraGain" : -1,
+  "cameraVideoModeIndex" : 0,
+  "streamingFrameDivisor" : "NONE",
+  "ledMode" : true,
+  "inputShouldShow" : true,
+  "outputShouldShow" : true,
+  "hsvHue" : {
+    "first" : 0,
+    "second" : 20
```

Comment:
Since the hue range for red wraps from like 190->20 let's try manually setting the hue range via this JSON. 
Here's what I mean
![image](https://user-images.githubusercontent.com/3326002/152269319-81999470-9dcc-4518-bdb5-5fd7f3220c31.png)
![image](https://user-images.githubusercontent.com/3326002/152269495-6bdc484d-f6e6-451a-9d21-a50acefa25f9.png)

You can see red hue range actually begins at the end, and wraps around to the beginning. I read that Photonvision supports this on the backend but haven't figured out how to do this via the UI. I've asked them in Discord to verify. Let's try this out.